kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/VendorCreditMemoDocument.java

```diff
@@ -114,12 +113,13 @@ public void prepareForSave() {
                 setPaymentPaidYear(null);
             }
         }
+
+        setPayment1099Indicator(false);
         for (CreditMemoIncomeType incomeType : getIncomeTypes()) {
-            if ((StringUtils.isBlank(incomeType.getIncomeTypeCode()) || incomeType.getIncomeTypeCode().equals(KFSConstants.IncomeTypeConstants.INCOME_TYPE_NON_REPORTABLE_CODE))) {
-                setPayment1099Indicator(false);
-            } else {
+            boolean isCodeExist = StringUtils.isNotBlank(incomeType.getIncomeTypeCode());
+            boolean isReportable = !incomeType.getIncomeTypeCode().equals(KFSConstants.IncomeTypeConstants.INCOME_TYPE_NON_REPORTABLE_CODE);
+            if (isCodeExist && isReportable) {
                 setPayment1099Indicator(true);
-                break;
```

Comment:
Ditto on the 'break' getting deleted here. In all three cases I've seen of this so far, each has been a generic mutator on  a solitary value (as opposed to a complex object) -- have I missed anything of why we would want to continue testing past the first positive hit? If I haven't, and given we have three examples of this happening, I think it might be worth tidying up.